### PR TITLE
Show Notification Action

### PR DESCRIPTION
**Pull request by ikaikastine**
**Checks**

- [x] I've checked there are no linting errors.
- [x] I've checked the code is still at 100% test coverage.

**Added Actions (if relevant)**

- Show Notification

**Are you happy to be listed as a contributor on [Shortcuts.fun](https://shortcuts.fun)?**

Yes

**Any other information / comments**

Process was really easy and easy to follow! Took me a little bit to get familiar with everything, but looking at other code helped me get up to speed. Contributing guide made sense and was easy to understand.

### CODE REVIEW

File: src/actions/showNotification.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ *  title: 'Hello, World!',
+ *  body: 'This is a test notification',
+ *  sound: true,
+ * });
+ * ```
+ */
+
+const showNotification = (
+  options: {
+    /* Title for the notification */
```
Comment (joshfarrant) [inline src/actions/showNotification.ts]: Could the start of this comment be changed to `/**` please? Also, could you add similar comments for `body` and `sound`, just explaining in a couple of words what the do. 🙂

**Comment by joshfarrant** [comment]
Hey @ikaikastine 👋 Great contribution, thanks! Just one tiny change then it's good to go 🎉

File: src/actions/showNotification.ts (hunk 1 above)
Comment (ikaikastine) [reply src/actions/showNotification.ts]: Comment format updated & comments added for body and sound!

**Comment by joshfarrant** [comment]
Fantastic, thanks!

**Pull request merged**